reliability-engineering/terraform/modules/concourse-monitoring/grafana.tf

```diff
@@ -45,30 +45,30 @@ resource "aws_db_instance" "concourse_grafana_db" {
   instance_class            = "db.t2.small"
   name                      = "grafana"
   username                  = "grafana"
-  password                  = "${random_string.concourse_grafana_db_password.result}"
-  vpc_security_group_ids    = ["${aws_security_group.concourse_grafana_db.id}"]
-  db_subnet_group_name      = "${aws_db_subnet_group.concourse_grafana_db.name}"
+  password                  = random_string.concourse_grafana_db_password.result
+  vpc_security_group_ids    = [aws_security_group.concourse_grafana_db.id]
```

Comment:
@samcrang  Could this be??:
vpc_security_group_ids    = aws_security_group.concourse_grafana_db.id